src/bestof2016/comments/index.js

```diff
@@ -58,7 +58,7 @@ We have already mentioned {vuejs} (number 1 overall), let's see the other conten
 
 {react} is so popular that it inspired a lot of other libraries that aim to take the best of React, without the fat, in order to improve both performance in the browser and building time.
 
-{inferno} was the most popular project in this category, it claims to be the fatest alternative to React
+{inferno} was the most popular project in this category, it claims to be the faster alternative to React
```

Comment:
Here (maybe) it is better to use `... claims to be *a* faster alternative ...`
Not sure though...